formbuilder/components/builder/AddFieldDropdown.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import SchemaField from "react-jsonschema-form/lib/components/fields/SchemaField";
+
+import {Dropdown, MenuItem}  from "react-bootstrap"
+
+import config from "../../config";
+
+export default class AddFieldDropdown extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {'fieldList': config.fieldList};
+  }
+
+  handleAddField(field_index, event) {
+    const field = this.state.fieldList[parseInt(field_index)];
```

Comment:
i didn't see any indexError, with a wrong index i just get a undefined field.
it that would be ok to add a test on undefined field in reducer/form.js addField ?